messages/ais/vessel.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Some comments on this:
* ROT?
* Navigational status?
* Message 5 needs a separate spec?

Reply:
Should AIS position also be here? Due to the AIS position is dependent on selected GNSS unit connected or integrated